neo/core/imagesequence.py

```diff
@@ -0,0 +1,76 @@
+from neo.core.regionofinterest import RegionOfInterest
+from neo.core.analogsignal import AnalogSignal, _get_sampling_rate
+from neo.core.dataobject import DataObject
+import quantities as pq
+import numpy as np
+from neo.core.baseneo import BaseNeo
+
+
+class ImageSequence(DataObject):
```

Comment:
I think it would be better to inherit from `BaseSignal` instead of from `DataObject`.